src/main/resources/archetype-resources/angular-app/src/main.ts

```diff
@@ -21,7 +21,15 @@ import { environment } from './environments/environment';
 
 if (environment.production) {
   enableProdMode();
+} else {
+  // On localhost (dev server), redirect from "/" to app root
+  const isLocalhost =
+    location.hostname === 'localhost' || location.hostname === '127.0.0.1';
```

Comment:
See comment below for index.html - I think NODE_ENV is available in Angular as well, though have not checked.

Reply:
This shouldn't even be necessary with Angular, since different `environment.ts` files are included in development/production mode (see `fileReplacements`)